core/tests/org.openjdk.jmc.testlib/src/main/java/org/openjdk/jmc/test/MCTestCase.java

```diff
@@ -30,9 +30,7 @@
  * WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
  * WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package org.openjdk.jmc.common.test;
-
-import static org.junit.Assert.assertArrayEquals;
+package org.openjdk.jmc.test;
```

Comment:
Hm, that would be possible but it would render the change a lot larger. There are also subpackages `io` and `mock.item`. If we were to rename all 3 packages we'd probably have to touch more tests. So, if we opt to rename the packages I'd prefer to do it in a separate PR.

Reply:
Ok, I understand the concern.